Source/Helpers/StringSplitEnumerator.cs

```diff
@@ -35,7 +35,7 @@ public bool MoveNext()
                     return true;
                 }
 
-                var slice = span.Slice(0, index).Trim();
+                var slice = span.Slice(0, index);
                 _str = span.Slice(index + 1);
                 if (slice.Length == 0)

```

Comment:
https://github.com/svg-net/SVG/blob/5dc4460eaf9819909b8a3beb993f3ba4eb7dbf6e/Source/Helpers/StringSplitEnumerator.cs#L38-L44

The following code is more efficient.
```cs
    _str = span.Slice(index + 1);
    if (index == 0)
        continue;
    Current = new StringPart(span.Slice(0, index));
```